### PR TITLE
Global antora file

### PR DESCRIPTION
**Pull request by fantkolja**
Should be merged after https://github.com/hazelcast/hazelcast-docs/pull/412

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -17,7 +17,18 @@ jobs:
...
         node-version: 16
+
+    - name: Checkout global antora-playbook
+      uses: actions/checkout@v4
+      with:
+        repository: hazelcast/hazelcast-docs
+        sparse-checkout: antora-playbook.yml
+        sparse-checkout-cone-mode: false
+        path: hazelcast-docs
+        ref: main
```
Comment (JackPGreen) [inline .github/workflows/validate.yml:28]: What happens if someone opens a PR on a non-`main` branch in this repo?

File: package.json [hunk 2]
```diff
@@ -7,17 +7,19 @@
...
     "check-links-local": "antora --fetch --to-dir test --log-level=error --log-failure-level=error --extension=@jcahills/antora-link-checker check-links-playbook.yml",
     "serve": "serve test",
     "expose": "ngrok http 5000"
   },
   "devDependencies": {
     "@antora/cli": "^3.1.1",
     "@antora/site-generator": "^3.1.1",
-    "@jcahills/antora-link-checker": "^1.0.1",
     "@djencks/antora-aggregate-collector": "^0.1.0-beta.1",
+    "@jcahills/antora-link-checker": "^1.0.1",
```
Comment (JackPGreen) [inline package.json:19]: Turbo-nit - this is the same, just re-ordered?

File: check-links-playbook.yml
Comment (JackPGreen) [inline check-links-playbook.yml:1]: Should we also delete the definition from `package.json`?

File: load-check-links-playbook.js
Comment (JackPGreen) [inline load-check-links-playbook.js:1]: _If_ the plan is to share this with other repos, can we centralise it?

Comment (JackPGreen) [reply load-check-links-playbook.js:1]: Some summary documentation would help on what this does at a high-level.

**Review by JackPGreen — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fantkolja) [reply package.json:19]: Yes, just got reordered alphabetically , after I've launched `npm i`

File: check-links-playbook.yml
Comment (fantkolja) [reply check-links-playbook.yml:1]: The script [load-check-links-playbook.js](https://github.com/hazelcast/management-center-docs/pull/386/files#diff-ab850f5aa851d60af16ecfb679c1121f0977b511ba533ba919da38c7a640e1c7R44) creates that `check-links-playbook.yml` but on the fly.

But I also don't like that `check-links-local` references a non-existing file, so I will probably refactor this part in the further work...

File: load-check-links-playbook.js
Comment (fantkolja) [reply load-check-links-playbook.js:1]: Will do both: centralizing and docs after we approve that the whole idea is worth implementing

File: .github/workflows/validate.yml (hunk 1 above)
Comment (JackPGreen) [reply .github/workflows/validate.yml:28]: 🤦

**Review by JackPGreen — APPROVED** [review]

**Pull request merged**